x/group/server/msg_server.go

```diff
@@ -98,6 +97,9 @@ func (s serverImpl) UpdateGroupMembers(ctx types.Context, req *group.MsgUpdateGr
 			return err
 		}
 		for i := range req.MemberUpdates {
+			if err := assertMetadataLength(req.MemberUpdates[i].Metadata, s.maxMetadataLength(ctx), "group member metadata"); err != nil {
```

Comment:
Not related to this PR, just curious: why is `maxMetadataLength` a function which takes sdk.Context, and not just a const?

Reply:
this max metadata length used to be part of the `params` and this hasn't been cleaned up, will do that.